website/app/components/doc/page/header/index.hbs

```diff
@@ -1,5 +1,5 @@
 <header class="doc-page-header" ...attributes>
-  <NavigationNarrator />
+  <NavigationNarrator @routeChangeValidator={{@routeChangeValidator}} />
```

Comment:
I _think_ that, since the `transitionValidator` function is declared in the `application` controller, then it's available here as `this.transitionValidator` (in which case you don't need to propagate/pass it down via an argument). Can you check?

Reply:
I tried that the first time and it didn't work, unfortunately. I'll give it another go, maybe a missed something.